### ISSUE TITLE
When *changing* Main Stage Action unable to select "Image" or "Map"

### ISSUE THREAD

**Issue by martin-martin** [Esri/map-journal-storytelling-template-js#65]
**Unable to select "Image" when changing the main stage action image to a different image.**

1. After selecting text and choosing "Change the Main Stage Content", the modal defaults to "Map".
    - "Video" and "Web Page" can be selected, however "Image" not.
2. After selecting any of the other, also "Map" cannot be selected anymore.
    - Hovering on map changes the cursor to the hand symbol.

---

_see Screenshots for details_

selecting text:
![screen shot 2016-06-22 at 11 26 32](https://cloud.githubusercontent.com/assets/12371494/16278906/82ed6542-386d-11e6-8c91-eef5e9ee3225.png)

"Map" is default setting:
![screen shot 2016-06-22 at 11 26 54](https://cloud.githubusercontent.com/assets/12371494/16278938/9f7e7b4c-386d-11e6-921a-1d8ecc0e85be.png)

after selecting any other, both "Map" and "Image" become inaccessible:
![screen shot 2016-06-22 at 11 27 06](https://cloud.githubusercontent.com/assets/12371494/16278941/a65d4eb6-386d-11e6-928f-e28392e3b7b1.png)

**Comment by glazou** [comment]
Hi,
Thanks for reporting this issue. I was looking to fix this but I cannot reproduce. It may be due to something specific about your story. I can tell from the style of the button that you are on a Mac. I tried to reproduce on Chrome and Safari. Can you try to reproduce it on a new story and let me know?

**Comment by martin-martin** [comment]
Hei glazou.

I've tried to reproduce the glitch with a new storymap, and also didn't come to the same result.

The difference that I noticed is, that I cannot select anything from the "Main Stage Actions". So it seems I now simply don't get to the point where the error occurred.

That's a good thing maybe? : )

**Comment by martin-martin** [comment]
Hei glazou!

I've tried it out now with a different storymap project and the error doesn't seem to be present!

Yay! : )
Not sure what I did differently last time, sorry.
But I remember that I tried a few times and the issue kept showing up (I think I even showed David, asking whether this was an issue...)

Well, either I messed something up back then or the bug got resolved, it seems.
So all is well : )

Thanks for following up on this.
